packages/inbound-email-handler/src/newsletter.ts

```diff
@@ -6,9 +6,12 @@ const EMAIL_CONFIRMATION_CODE_RECEIVED_TOPIC = 'emailConfirmationCodeReceived'
 const EMAIL_FORWARDING_SENDER_ADDRESSES = [
   'Gmail Team <forwarding-noreply@google.com>',
 ]
-const NEWSLETTER_SENDER_REGEX = '<.+@axios.com>'
-const CONFIRMATION_CODE_PATTERN = '^\\(#\\d+\\)'
-const AXIOS_URL_PATTERN = 'View in browser at <.+>'
+const NEWSLETTER_SENDER_REGEX =
```

Comment:
Eventually we will need a more scalable solution here. I wonder if there are some more headers we can look at.

Reply:
Yeah I agree. Unfortunately, Bloomberg and Axios newsletters are unlike Substack's which provides very informative headers. So we can only either check the email sender address or parse the DOM